src/Ng/Logging.cs

```diff
@@ -0,0 +1,244 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Runtime.Remoting;
+using System.Runtime.Remoting.Messaging;
+using Microsoft.Extensions.Logging;
+using Serilog;
+using Serilog.Core;
+using Serilog.Events;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace Ng
+{
+    internal static class Logging
```

Comment:
Both svc and jobs now share exact same logging code, fixed in https://github.com/NuGet/NuGet.Services.Metadata/commit/e5479a79105e4e73016730dcefddf035071a4e31 which also adds basic auth support against the ES cluster.